### PR TITLE
Fetch service versions from Marathon

### PR DESCRIPTION
**Pull request by orlandohohmeier**
Add actions and methods to fetch and parse service versions from Marathon.

### CODE REVIEW

**Comment by mesosphere-web** [comment]
Reviewer: @rcorral

File: src/js/events/__tests__/MarathonActions-test.js [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+
+var ActionTypes = require('../../constants/ActionTypes');
+var AppDispatcher = require('../AppDispatcher');
+var Config = require('../../config/Config');
+var RequestUtil = require('../../utils/RequestUtil');
+var MarathonActions = require('../MarathonActions');
+
+describe('MarathonActions', function () {
+
+  describe('#fetchGroups', function () {
```
Comment (mlunoe) [inline src/js/events/__tests__/MarathonActions-test.js]: I guess these were missing?

Comment (orlandohohmeier) [inline src/js/events/__tests__/MarathonActions-test.js]: Yep.

**Comment by mlunoe** [comment]
This looks good to me! Good work @orlandohohmeier!

File: src/js/events/MarathonActions.js [hunk 2]
```diff
@@ -47,6 +51,43 @@ module.exports = {
       };
     },
     {delayAfterCount: Config.delayAfterErrorCount}
-  )
+  ),
+
+  fetchServiceVersion: function (serviceId, versionId) {
```
Comment (kennyt) [inline src/js/events/MarathonActions.js]: can we name these `serviceID` and `versionID`? we usually capitalize the `ID`

File: src/js/events/__tests__/MarathonActions-test.js [hunk 3]
```diff
@@ -0,0 +1,158 @@
...
+      });
+
+      this.configuration.error({message: 'error'});
+    });
+
+  });
+
+  describe('#fetchServiceVersion', function () {
+
+    const serviceId = 'test';
```
Comment (kennyt) [inline src/js/events/__tests__/MarathonActions-test.js]: `serviceID`/`versionID`

**Comment by kennyt** [comment]
feel free to merge once variables are changed

**Comment by pierlo-upitup** [comment]
LGTM! 🚢

**Pull request merged**